assets/js/components/HostDetails/HostSettingsPage.jsx

```diff
@@ -49,6 +56,19 @@ function HostSettingsPage() {
           })
         )
       }
+      saving={saving}
+      error={savingError}
+      success={savingSuccess}
+      onSave={(selection, targetID) =>
+        dispatch(
+          checksSelected({
+            targetID,
+            targetType: 'host',
+            targetName: host.hostname,
```

Comment:
Getting this `host` value from outside the function itself looks a bit risky, doesn't it?

Reply:
What do you mean by risky? The fact that a host might not be loaded yet?
If that is the case we should be fine with the early return of a loading box if we don't have a host on line 30

Reply:
Correct that is the intended evolution providing targetID, targetName and targetType.